jobs.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Compiler Jobs
+---
+<h1>Jobs</h1>
+
+<h2>Accepting applications</h2>
```

Comment:
I'd like this to be a little smarter based on having (open) positions - e.g. here we would render an unnecessary `<ul>` with no `<li>` if there are no open positions.

Jekyll (Liquid) has a filter for collections based on a property. The way this works is you create a new variable and assign it the filtered results:

```liquid
{% assign open_jobs = site.jobs | where: "open" %}
```

So that's grabbing all `site.jobs` with a "truthy" `open` attribute. 

Then you can do something like:

```liquid
{% if open_jobs.size > 0 %}
  # render links
{% else %}
  # render "We aren't accepting applications at the moment, please email us your resume" etc.
```

Read more about Liquid (the templating language under Jekyll): https://shopify.github.io/liquid/

Reply:
Thanks! I like this idea.

I ran into some trouble with trying to use `where: "open"`. I kept getting this error from `jekyll build`:
```
Liquid Exception: Liquid error (line 4): wrong number of arguments (given 2, expected 3) in jobs.html
             Error: Liquid error (line 4): wrong number of arguments (given 2, expected 3)
             Error: Run jekyll build --trace for more information.
```
The [Jekyll documentation on filters](https://jekyllrb.com/docs/liquid/filters/) doesn't mention or show examples of this "truthy"-`where`, so I ended up using `where: "open", "true"` in 91d81a82b93a4f77dfc742239219b206bdede49e to effectively get the same result.

I read in the [Liquid documentation](https://shopify.github.io/liquid/basics/variations/) that Jekyll "might not be using the latest version of Liquid", so I think that's probably why the truthy-`where` isn't supported.